helm/boutique/templates/_helpers.tpl

```diff
@@ -51,6 +96,9 @@
     httpGet:
       path: /health
       port: http
+  volumeMounts:
+    - name: apmsocketpath
+      mountPath: /var/run/datadog
```

Comment:
I am wondering the pros/cons of

https://docs.datadoghq.com/containers/kubernetes/apm/?tab=unixdomainsocketuds
vs
https://docs.datadoghq.com/containers/kubernetes/apm/?tab=tcp

ps: I used the latter in my configuration